src/main/query_profiler.cpp

```diff
@@ -122,15 +123,7 @@ void QueryProfiler::EndQuery() {
 	auto automatic_print_format = GetPrintFormat();
 	// print or output the query profiling after termination, if this is enabled
 	if (automatic_print_format != ProfilerPrintFormat::NONE) {
```

Comment:
Could we perhaps turn this if into `if (!is_explain_analyze)`, and remove the `ProfilerPrintFormat::NONE` option entirely?